Source/player.cpp

```diff
@@ -1047,7 +1047,7 @@ bool PlrHitPlr(int pnum, int8_t p)
 		return false;
 	}
 
-	if ((target._pSpellFlags & 1) != 0) {
+	if ((target._pSpellFlags & SF_ETHEREALIZE) != 0) {

```

Comment:
This makes it clear what need to be remove :D

Reply:
haha yeah that's what I figured 😄